recipes/nvidia-tao-pytorch/recipe.yaml

```
@@ -0,0 +1,65 @@
+schema_version: 1
+
+context:
+  name: nvidia-tao-pytorch
+  version: 6.25.7
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  # Need to use hash because tags are missing, see https://github.com/NVIDIA/tao_pytorch_backend/issues/41
+  url: https://github.com/NVIDIA/tao_pytorch_backend/archive/70a3d9128bcf4a4675e6a1d1cf763df7b1803b3e.tar.gz # v6.25.7
+  sha256: d8e02b26b1d5defce139c9459ea1def3e8659916e26b665f3fce8d13cc1839c8
+
+build:
+  number: 0
+  script:
+    env:
+      TORCH_CUDA_ARCH_LIST: "7.5 8.0 8.6 8.9 9.0+PTX"
```

```suggestion
      TORCH_CUDA_ARCH_LIST: "5.0 6.0 7.0 7.5 8.0 8.6 9.0+PTX"
```
This works instead. We can add 10.0 and 12.0 once we switch over to CUDA 12.9 in the feedstock.